src/core/api/public_api.ts

```diff
@@ -1791,19 +1800,27 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     let trackId : string;
     let switchingMode : IVideoTrackSwitchingMode | undefined;
     let reprsToLock : string[] | null = null;
+    let relativeResumingPosition : number | undefined;
     if (typeof arg === "string") {
       trackId = arg;
     } else {
       trackId = arg.trackId;
       periodId = arg.periodId;
       switchingMode = arg.switchingMode;
       reprsToLock = arg.lockedRepresentations ?? null;
+      relativeResumingPosition = arg.relativeResumingPosition;
     }
     return this._priv_callTracksStoreGetterSetter(
       periodId,
       undefined,
       (tcm, periodRef) =>
-        tcm.setVideoTrack(periodRef, trackId, switchingMode, reprsToLock));
+        tcm.setVideoTrack({
+          periodObj: periodRef,
+          wantedId: trackId,
+          switchingMode,
+          reprsToLock,
+          relativeResumingPosition,
```

Comment:
You forgot to also update this one (which now has a different API than `tcm.setAudioTrack`)

Reply:
thanks, I'm adding it